demo/app/ios/Runner/Tests/IntegrationTest.swift

```diff
@@ -69,7 +69,7 @@ class IntegrationTest: XCTestCase {
         let vpInteraction = Openid4vpNewInteraction(openID4VPArgs, opts, nil)!
 
         let credentialsQuery = try vpInteraction.getQuery()
-        let inquirer = CredentialNewInquirer(nil)!
+        let inquirer = CredentialNewInquirer(nil, nil)!
```

Comment:
Don't ignore the error; need to check it

Reply:
We always ignore errors like this. So we need to handle all of them in tests or ignore them like previously.

Reply:
It's a mistake to ignore the errors. We need to at least start adding error checks for new code.

Reply:
Perhaps sometime in the future we can add in checks for the other existing errors (not in this PR of course)

Reply:
done